plugins/statistics.koplugin/main.lua

```diff
@@ -2416,7 +2416,8 @@ function ReaderStatistics:onSaveSettings()
 end
 
 -- in case when screensaver starts
-function ReaderStatistics:onSuspend()
+function ReaderStatistics:onSuspend(suspend_to)
+    if suspend_to == "standby" then return end
```

Comment:
Not really wanting to get involved with the auto* plugins, so not really looking at this PR core stuff.
But from an outsider point of view, why are Resume/Suspend events sent, if they are not real Resume/Suspend ?
Do all receivers having onSuspend/onResume handlers should get added this kludge of `if suspend_to == "standby" then return end` ?

Reply:
Good point. Of course, it would be possible to have two additional events `onEnterStandby` and `onLeaveStandby`, which would be sufficient in most cases. 
In the other cases (mainly readerfooter autoupdate and autoturn) there is a bit of work to switch to the new events. 

But I think it might be clearer for future development, to have one "big" event for entering and one event for leaving power saving mechanisms. Then the developer can focus on both (standby and suspend) together. In some standby cases it is not important, that the clock freezes during standby, whereas in other cases it is necessary to know the "lost time".

Maybe a renaming of `onSuspend` and `onResume` to `onEnterPowerSaving` and `onLeavePowerSaving` might be better then. But let's postpone that decision to later. Currently, there are only two handful of `onSuspend` and `onResume` handlers in the code.

Reply:
> But let's postpone that decision to later. 

Well, given that the ReaderStatistics plugin should not care about onEnter/LeavePowerSaving, I'd rather you not postpone it, and have this PR not touch unrelated things :)

Reply:
I'd rather not make the names any more obscure than they already are, though. "PowerSaving" doesn't make anything. Suspend does. (Standby.. doesn't, but we haven't thought of anything better ;p).

Reply:
I'm not against doing the opposite:

splitting the events in four (or three?). Plugins that don't care about the distinction can just set both handlers to the same function.